RNA-seq/03-gastric_cancer_exploratory-live.Rmd

```diff
@@ -27,23 +27,16 @@ from Love, Anders, and Huber from which this is adapted
 ```
 
 ```{r magrittr}
-# magrittr for the pipe
```

Comment:
I am confused why `git checkout master RNA-seq/03-gastric_cancer_exploratory-live.Rmd` does not restore this but instead tells me 0 paths are updated.

Reply:
I still don't understand why this command wasn't working but I just did a manual copy and paste from this file from the master branch and that worked.